build/grunt.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(grunt) {
       release: {
         tag_name: 'v'+ version.full,
         name: version.full,
-        body: nodeCli.exec('conventional-changelog', '-p videojs', {silent: true}).output
+        body: npmRun.execSync('conventional-changelog -p videojs', {silent: true})
```

Comment:
This needs an `encoding: 'utf8'` options because a buffer is returned by default.